MainModule/Server/Core/HTTP.lua

```diff
@@ -38,7 +38,11 @@ return function(Vargs)
 	server.HTTP = {
 		Init = Init;
 		Service = service.HttpService;
+<<<<<<< Updated upstream
 		HttpEnabled = pcall(service.HttpService.GetAsync, service.HttpService, "https://google.com");
+=======
+		HttpEnabled = pcall(service.HttpService.GetAsync, service.HttpService, "http://www.google.com/robots.txt");
+>>>>>>> Stashed changes
```

Comment:
Yet more jargon

Reply:
That is by Github, not me. Lemme fix that right quick